proxy/Dockerfile

```diff
@@ -43,8 +43,9 @@ RUN if [ -n "$PROXY_UNOPTIMIZED" ]; \
     fi
 
 # Build gRPC bindings, leaving the proxy mocked out.
-COPY proto                  proto
-COPY proxy/controller-grpc  proxy/controller-grpc
+COPY proto/proxy proto/proxy
+COPY proto/common proto/common
```

Comment:
Innnnteresting -- it looks like `proto/common` imports the google protobuf duration file, but you're not copying `proto/google` here and the proxy still builds successfully. I can't seem to figure out why that file isn't required for the proxy. It's definitely required when building the go bindings.

Reply:
The protobuf binding stuff for rust, the "prost" crate, includes its own copies of the "google" protobufs.

Reply:
Ah, yep, that would do it. Thanks for the explanation!